lib/driver.js

```diff
@@ -647,14 +646,6 @@ class AndroidUiautomator2Driver extends BaseDriver {
         })();
       }));
 
-      if (this.opts.unicodeKeyboard && this.opts.resetKeyboard && this.defaultIME) {
-        this.log.debug(`Resetting IME to '${this.defaultIME}'`);
-        try {
-          await this.adb.setIME(this.defaultIME);
-        } catch (err) {
-          this.log.warn(`Unable to reset IME: ${err.message}`);
-        }
-      }
       if (this.caps.androidCoverage) {
```

Comment:
we should also probably get rid of these caps.
I've never seen them used anywhere